internal/pkg/cli/app_package.go

```diff
@@ -107,7 +115,19 @@ func (opts *PackageAppOpts) Validate() error {
 
 // Execute prints the CloudFormation template of the application for the environment.
 func (opts *PackageAppOpts) Execute() error {
-	return nil
+	project := viper.GetString(projectFlag)
```

Comment:
Should there be a `--project`/`-p` flag either on the `app` sub-command or even root command where we bind the viper value to that flag? I think that should help remove these `viper.GetString(projectFlag)` throughout the codebase. I can't think of too many situations where a project name will not need to be in context.

Reply:
Ok this is what I'm thinking, y'all tell me if you like it:

1. We'll move the `bindProjectName()` function to `init` function in the `internal/cli` pkg.
2. All the `Opts` structs will have a private `projectName` field that's set to `viper.GetString("project")` in their constructor.

This way it will allow us to write tests without having the set the field with viper.

I don't think I see a con w/ this approach.

Reply:
Could we bind the value to a `GlobalOpts` struct and embed that in sub-command `Opts`? I'm trying to think of a way to avoid the duplicate definitions so that if we add another "global option" it will just be available to sub-commands that rely on that structure.

Reply:
Good idea! Done :)